src/templates/admin/core/manager/users/history.html

```diff
@@ -6,7 +6,7 @@
     {{ block.super }}
     <li><a href="{% url 'core_manager_index' %}">Manager</a></li>
     <li><a href="{% url 'core_manager_users' %}">Users</a></li>
-    <li>History</li>
+    <li>History of {{user.first_name}} {{user.last_name}} </li>
```

Comment:
This time `user` is the right variable because the default Django `user` is overwritten in the view's `context` with the appropriate account:

https://github.com/BirkbeckCTP/janeway/blob/dac0e5a703b5a68f67946b23cd385df90bf959e0/src/core/views.py#L1241-L1273